File: Second Year (2013-2014)/OMGL7/TP2 - Website in PHP/musiconline/musiconline/db/Album.php
<?php
/* Authors: ANDRIAMILANTO Tompoariniaina && CANNO Camille
* Groupe: B1
* File: Album.php
*/

class Album implements DataObject{
   private $IdAlbum;
   private $LibelleAlbum;

   public function  __construct() {}

   public function init($IdAlbum,$LibelleAlbum){
      $this->IdAlbum=$IdAlbum;
      $this->LibelleAlbum=$LibelleAlbum;
   }


   public function getIDAlbum(){
     return $this->IdAlbum;
   }

   public function setIDAlbum($IdAlbum){
     $this->IdAlbum = $IdAlbum;
   }


   public function getLIbelleAlbum(){
     return $this->LibelleAlbum;
   }

   public function setLIbelleAlbum($LibelleAlbum){
     $this->LibelleAlbum = $LibelleAlbum;
   }


}
?>

File: Second Year (2013-2014)/OMGL7/TP3 - Website in JSP/src/db/Album.java
package db;
/**
*@author Andriamilanto Tompoariniaina - Canno Camille - Info 2 B
*/

/**
*Classe pour les objets Album
*/
public class Album {
//Attributs
  private int idAlbum;
  private String libelleAlbum;

//Methodes
  /**
  *Constructeur
  */
  public Album() {}

  /**
  *Methode pour initialiser les attributs
  */
  public void init(int idA, String lib) {
    this.idAlbum = idA;
    this.libelleAlbum = lib;
  }

//Accesseurs
  public void setIdAlbum(int idA) {
    this.idAlbum = idA;
  }

  public int getIdAlbum() {
    return this.idAlbum;
  }

  public void setLibelleAlbum(String libA) {
    this.libelleAlbum = libA;
  }

  public String getLibelleAlbum() {
    return this.libelleAlbum;
  }
}
